heralding/capabilities/socks5.py

```diff
@@ -63,6 +64,23 @@ class Socks5(HandlerBase):
         else:
             logger.debug("Wrong authentication data: %r" % auth_data)
 
+    def get_auxiliary_data(self, authmethods):
```

Comment:
Could we also log the socks5 version? As far as i read the code, the client also transmits a value containing  some kind of versioning?

Reply:
The client transmits the Socks version. Here it should be always 5. If, its not 
the line 65 executes.